universum/analyzers/uncrustify.py

```diff
@@ -29,15 +33,17 @@ def main() -> int:
     files: List[Tuple[Path, Path]] = []
     try:
         src_files: List[Path] = _get_src_files(settings.file_list, settings.file_list_config)
+        if not src_files:
+            raise EnvironmentError("Please provide at least one file for analysis")
```

Comment:
Why not use Universum-specific exceptions, such as 'CiException' or 'CriticalCiException'?